src/state/REPLState.js

```diff
@@ -40,6 +40,8 @@ function decodeBase64(base64String) {
 }
 
 class REPLState {
+  static baseURL = "http://localhost:1337";
```

Comment:
Do you think we should set the baseURL in `package.json` by adding a proxy? Ex.:
```
"proxy": "http://localhost:1337"
```

Reply:
I didn't know this was a thing! Go for it

Reply:
@mohammedsahl is also looking at bundling the client and server